package-lock.json

```diff
@@ -1971,6 +1971,11 @@
       "integrity": "sha512-SEQu7vl8KjNL2eoGBLF3+wAjpsNfA9XMlXAYj/3EdaNfAlxKthD1xjEQfGOUhllCGGJVNY34bRr6lPINhNjyZw==",
       "dev": true
     },
+    "i": {
+      "version": "0.3.6",
+      "resolved": "https://registry.npmjs.org/i/-/i-0.3.6.tgz",
+      "integrity": "sha1-2WyScyB28HJxG2sQ/X1PZa2O4j0="
+    },
```

Comment:
Next time, when removing a dependency run

`npm remove <dependency>` 

rather than modifying the package.json by hand so that the lockfile can be updated as well.

Reply:
Noted. Thanks again.